doc/release/0.4.rst

```diff
@@ -24,7 +24,7 @@ API Changes
 #. Removing unused function ``SkippingXsGen_BuChangedLessThanTolerance``. (`PR#1845 <https://github.com/terrapower/armi/pull/1845>`_)
 #. Allow for unknown Flags when opening a DB. (`PR#1844 <https://github.com/terrapower/armi/pull/1835>`_)
 #. ``Assembly.rotatePins`` and ``Block.rotatePins`` have been removed. Prefer to use
-   :meth:`armi.reactor.assemblies.Assembly.rotate` and meth:`armi.reactor.block.Block.rotate`
+   :meth:`armi.reactor.assemblies.Assembly.rotate` and :meth:`armi.reactor.block.Block.rotate`
```

Comment:
```suggestion
  ``Assembly.rotate`` and ``Block.rotate``
```

Reply:
I know this is an RST file, but we use this file in many and varied ways, and not all of those ways will be built with Sphinx and the ARMI API docs handy for auto-linking.